CONTRIBUTING.es.md

```diff
@@ -0,0 +1,164 @@
+
+```markdown
+![Ferris, la adorable mascota crustáceo de Rust](./ferris.png)
+```
+
+>¡Asegúrese de agregar texto descriptivo! Piense en la información que esta imagen agrega a su publicación y qué contexto es importante para lectores/lectoras con problemas de visión.
+
+Los videos también pueden ser integrados usando la siguiente sintaxis:
+
+```html
+<video src="./ios_vs_android.mp4" titulo="Una comparacion sobre como se aplica el espaciado de texto entre iOS y Android"></video>
+```
+
+> Si es posible, de preferencia use elementos de tipo `<video>` sobre archivos tipo `.gif` u otras imagenes animadas en nuestras publicaciones. Esto es por preocupaciones relacionadas a la facilidad de acceso, ya que los videos le dan al usuario mas control sobre cuando y como se reproduce la animación.
+
+# Traducir Publicación de Blog
+
+¡Si usted agrega una traducción, asegúrese de primero crear un [Perfil de Autor](#crear-perfil-de-autor) con el rol de `traductor` para que sea acreditado por su trabajo en la página!
+
+Para crear una traducción para una publicación, copia el `index.md` del documento y cambia su nombre a `index.(lang).md`, donde `(lang)` es el idioma de la traducción. Por ejemplo, una traducción en `fr` (francés) debe ser nombrar `index.fr.md`. Hecho esto, se puede traducir el contenido de este documento a su respectivo idioma.
+
+> Imágenes en la publicación que necesiten traducción deben ser nombradas de la misma manera. Por ejemplo, una traducción en francés para `dom_tree.svg` se debe nombrar `dom_tree.fr.svg`.
+>
+> Todos los enlaces a estas imágenes necesitarán ser actualizados en la publicación `index.fr.md` para apuntar a la imagen traducida.
+
+Para referencia, los códigos de los idiomas actuales se pueden encontrar en [`/content/data/languages.json`](./content/data/languages.json). Se necesitará agregar el código del idioma si no está en este documento.
+
+## Buscando Código de Idioma
+
+Cada código de idioma en [`/content/data/languages.json`](./content/data/languages.json) debe consistir de dos letras minúsculas. Si incluye una región, agregue un guion seguido de otras dos letras minúsculas. Por ejemplo, el código del idioma francés es `fr`; para especificar el dialecto francés de Canadá, el código sería `fr-ca`.
+
+>Por favor, use `-` en lugar de `_` en los formatos ISO de la región del idioma. En lugar de `fr_ca`, seria `fr-ca`.
+
+Consulte [Wikipedia: List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) para conocer los identificadores que se utilizaran en este formato.
+
+# Enviar una solicitud de extracción
+
+Una vez que todos los cambios se hayan hecho, [crea una solicitud de extracción](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) para unir tu publicación al sitio.
```

Comment:
```suggestion
Una vez que todos los cambios se hayan hecho, [crea una solicitud de extracción](https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) para unir tu publicación al sitio.
```
Just found that GitHub also has a Spanish version of this page that can be linked here :)

Reply:
How do I implement these modifications? Do I have to do them manually or do I simply press the commit suggestion button?

Reply:
Yeah, the "commit suggestion" button will apply them!

The updates from [#867](https://github.com/unicorn-utterances/unicorn-utterances/pull/867/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055) would need to be done manually, though - it's mostly just a few sentences that were changed/added.